archlinuxcn/llvm-git/PKGBUILD

```diff
@@ -173,7 +173,7 @@ package_compiler-rt-git() {
 
     local _llvmver=$(echo "$pkgver" | grep -Po "(^[\.\d]+)")
 
-    _fakeinstall fakeinstall/usr/lib/clang/"$_llvmver"/lib/linux
```

Comment:
Why the change of path? The none git version seems to be installing to linux like the old version rather than CHOST.

Reply:
it on my system and other build hosts was saying that there was no such thing as blah blah blah, and i realized that it changed it's logic to $CHOST